pkg/webui/console/containers/gateway-location-form/gateway-location-form.js

```diff
@@ -30,7 +30,7 @@ const m = defineMessages({
   locationDescription: 'The location of this gateway may be publicly displayed',
   updateLocationFromStatus: 'Update from status messages',
   updateLocationFromStatusDescription:
-    'Update the location of this gateway based on incoming status messages',
+    'Update the location of this gateway based on incoming status messages (only works for authenticated gateway connections)',
```

Comment:
This should rather be a warning that appears when the checkbox is activated. Also, this should link to some documentation that explains the preconditions for the location updates to be functional in more detail.

@benolayinka @nejraselimovic do we have something like that already?

My suggestion to not stall this PR: remove this addition and open an issue about this.